README.md

```diff
@@ -41,13 +41,16 @@ Verify communication with the Redash API
 
     docker run -v $PWD/user-report.yaml:/home/automation/report.yaml -t $IMAGE --dry-run --verbose
 
-Running Unit Tests
-------------------
+Running Checks
+--------------
+
+To run lint and unit tests:
 
-    python3 -m venv ~/redashvenv1
-    source ~/redashvenv1/bin/activate
+    python3 -m venv ~/.venv/redash-email
```

Comment:
Yeah, that's probably a much better path name. :smile:

Reply:
Must have thought that it would be eaiser to reuse that virtual env, but more likely at setup for conflict!